src/components/atoms/table_tr_tasks.jsx

```diff
@@ -0,0 +1,40 @@
+import { useNavigate } from "react-router-dom";
+import { useState } from "react";
+import styled from "styled-components";
+import ModalWindow from "./ModalWindow";
+import Button from "./Button";
+
+const StyledTd = styled.td`
+  border: 1px solid #dddddd;
+  padding: 5px;
+  text-align: center;
+  font-size: 20px;
+`;
+
+const TableTr = ({ title, description, updatedAt, deadlineAt, id }) => {
+  const navigate = useNavigate();
+  const [isDestroyModalOpen, setDestroyModalOpen] = useState(false);
+  const routeChange = () => {
+    // navigate(`/tasks/${id1}`);
+    navigate(`/`);
+  };
+  return (
+    <>
+      <tr>
+        <StyledTd>{title}</StyledTd>
+        <StyledTd>{description}</StyledTd>
+        <StyledTd>{new Date(updatedAt).toLocaleDateString("RU-ru")}</StyledTd>
+        <StyledTd>{new Date(deadlineAt).toLocaleDateString("RU-ru")}</StyledTd>
+        <StyledTd>
+          <Button label="Go to task" onClick={routeChange(id)} color="thistle" />
```

Comment:
like that function called with first render, and in console warning
![image](https://github.com/Raily01/task_tracker_react/assets/51396910/45807da9-ab65-4f9e-b84b-96a75fd4365d)
```suggestion
          <Button label="Go to task" onClick={() => routeChange(id)} color="thistle" />
```